src/main/steps/common/common.content.ts

```diff
@@ -227,14 +226,20 @@ const getServiceName = (translations: typeof en, isDivorce: boolean): string =>
   return capitalize(serviceName);
 };
 
-const getSelectedGender = (userCase: Partial<CaseWithId>, isApplicant2: boolean): Gender => {
+const getSelectedGender = (userCase: Partial<CaseWithId>, isApplicant2: boolean): Gender | undefined => {
   if (isApplicant2 && userCase?.sameSex === Checkbox.Unchecked) {
-    return userCase?.gender === Gender.MALE ? (Gender.FEMALE as Gender) : (Gender.MALE as Gender);
+    if (userCase?.gender === Gender.MALE) {
+      return Gender.FEMALE;
+    } else if (userCase?.gender === Gender.FEMALE) {
+      return Gender.MALE;
+    } else {
+      return undefined;
```

Comment:
theres going to be places where its just going to say 'your undefined'

Reply:
no this method is not used outside of this file now, it's just used to generate the partner content, which takes into account undefined